src/System.Collections.NonGeneric/src/System/Collections/SortedList.cs

```diff
@@ -71,7 +71,7 @@ public class SortedList : IDictionary, ICloneable
         private KeyList keyList; // Do not rename (binary serialization)
         private ValueList valueList; // Do not rename (binary serialization)
         [NonSerialized]
-        private object _syncRoot;
+        private object _syncRoot; // Serialized indirectly in SyncSortedList
```

Comment:
It's not. It does however mean that if you change the type of it, the binary format of SyncSortedList changes, causing our tests to break. I am not sure whether that would break cross version serialization. See note above.

Reply:
And of course if you stored something in there that wasn't binary serializable, it would be a break.

Reply:
> I am not sure whether that would break cross version serialization. 

I do not think it would break it.

Reply:
OK, but it would if I changed SortedList to put something unserializable in here, so a comment seemed useful.

Reply:
Alright will do

Reply:
Mind if I do that in a follow up? Updating the binary blobs causes many other updates that I don't understand yet. I need to talk to @ViktorHofer

Reply:
ok